src/test/java/com/google/api/codegen/testdata/discoveries/foo.v1.json.overrides

```diff
@@ -4,5 +4,27 @@
   },
   "js|python": {
     "discoveryDocUrl": "localhost:8080/$discovery/foo?version=v1"
+  },
+  "csharp|go|java": {
+    "methods": {
+      "foo.bar.get": {
+        "fields": {
+          "foobar": {
+            "defaultValue": "\" \""
```

Comment:
Actually I'll just bail on this part of the PR for now, might as well do the whole thing later. There are other issues with default values anyway.